examples/one_degree_simulation.jl

```diff
@@ -256,7 +256,7 @@ end
 # Finally, we plot a snapshot of the surface speed, temperature, and the turbulent
 # eddy kinetic energy from the CATKE vertical mixing parameterization as well as the
 # sea ice speed and the effective sea ice thickness.
-fig = Figure(size = (1200, 1200))
```

Comment:
why this?

Reply:
This is what it looks like: 5 panels, squished in a very disturbing way

![Uploading image.png…]()